src/yaml/parser.cr

```diff
@@ -93,6 +98,14 @@ class YAML::Parser
     value
   end
 
+  private def merge_mapping!(mapping, key, value)
```

Comment:
It seems this method is never used? Maybe you refactored it a bit and forgot to remove or finish something?